alfresco-transform-core-aio/alfresco-transform-core-aio/src/test/java/org/alfresco/transformer/AIOTransformRegistryTest.java

```diff
@@ -280,7 +280,7 @@ private void transformTextAndCheckPageLength(int pageLimit) throws Exception
         int cutoff = pageLimit * pageLength;
         for (int i = 1; i <= lines; i++)
         {
-            sb.append(i);
+            sb.append(Integer.toString(i));
             sb.append(" I must not talk in class or feed my homework to my cat.\n");
```

Comment:
Error found while working on TextToPdfContentTransformerTest which had the same problem.